AK/SIMDExtras.h

```diff
@@ -226,7 +226,7 @@ ALWAYS_INLINE static T shuffle_or_0_impl(T a, Control control, IndexSequence<Idx
     // 1. Set all out of bounds values to ~0
     // Note: This is done so that  the optimization mentioned down below works
     // Note: Vector compares result in bitmasks, aka all 1s or all 0s per element
-    control |= ~((control > 0) | (control < N));
+    control |= ~((control >= 0) & (control < N));
```

Comment:
Not sure if https://github.com/LadybirdBrowser/ladybird/pull/794/files#diff-d88bf409610a6e0479f5f504f954cf0317850db1042dbcb14f352081f9004b25R223 needs the same treatment.

Reply:
@Hendiadyoin1 I think this was your change?

Reply:
@dzfrias how about leaving a comment around it so we can revisit it later if needed?

Reply:
Ok, sounds good!